quizzes/09_arrow.js

```diff
@@ -0,0 +1,160 @@
+console.log(
+  // curryAdd(2)(4)(6),
+  // genSeries(),
+  // genSeries(7, 100),
+  // shift("havin pasta for brunch"),
+  // shift("havin pasta for brunch", 13),
+  // shift("univa cnfgn sbe oehapu", 13),
+  // getPopulation().display()
+)
+
+function curryAdd(a) {
+  // TODO: refactor this function
+  // rewrite with arrow syntax
+  return function(b) { // eslint-disable-line func-names
+    return function(c) { // eslint-disable-line func-names
```

Comment:
Instead of inline eslint comments here, put a single comment at the bottom (like in `07_parameters.js`). Want to remove as many distractions as possible here :)